### PR TITLE
Bugs

### PR DESCRIPTION
**Pull request by shaqsingh**
I fixed a bunch of things in the issue queue. most of the commit messages reference the related issue

### CODE REVIEW

File: _includes/footer.html [hunk 1]
```diff
@@ -1,5 +1,13 @@
...
-    This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.<br />
-    <strong><a class="mt3 pa3" href="https://envirodatagov.org">The Environmental Data &amp; Governance Initiative, 2017</a></strong>
+<footer class="flex w-100 mid-gray mb1 ph4 pv2 f6">
+    <div id="footer_cc">
+        <small>This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.</small>
+    </div>
+    <div id="footer_edg">
+        <small class="mt2 pr3"><a class="black" href="https://envirodatagov.org">The Environmental Data &amp; Governance Initiative, 2017</a></small>
+        <div>
+            <a href="https://twitter.com/EnviroDGI" target="_blank" rel="noopener"><img class="social" src="/assets/tw.svg/"/></a>
```
Comment (dcwalk) [inline _includes/footer.html]: drop trailing `/` from each social media icon, they currently aren't loading

File: _includes/footer.html [hunk 2]
```diff
@@ -1,5 +1,13 @@
-<footer class="dib w-100 mid-gray tc mt4 mb1 pv2 f6">
-    <a class="ma4 pa3" rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons Licence" style="border-width: 0;" src="https://licensebuttons.net/l/by-nc-sa/4.0/80x15.png" /></a><br />
-    This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.<br />
-    <strong><a class="mt3 pa3" href="https://envirodatagov.org">The Environmental Data &amp; Governance Initiative, 2017</a></strong>
+<footer class="flex w-100 mid-gray mb1 ph4 pv2 f6">
+    <div id="footer_cc">
+        <small>This work is licensed under a <a rel="license" href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.</small>
```
Comment (dcwalk) [inline _includes/footer.html:3]: with the social media stuff this small now seems disproportionately small

File: _includes/footer.html [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+    </div>
+    <div id="footer_edg">
+        <small class="mt2 pr3"><a class="black" href="https://envirodatagov.org">The Environmental Data &amp; Governance Initiative, 2017</a></small>
+        <div>
+            <a href="https://twitter.com/EnviroDGI" target="_blank" rel="noopener"><img class="social" src="/assets/tw.svg/"/></a>
+            <a href="https://www.facebook.com/EnviroDGI/" target="_blank" rel="noopener"><img class="social" src="/assets/fb.svg/"/></a>
+            <a href="https://www.instagram.com/envirodgi/" target="_blank" rel="noopener"><img class="social" src="/assets/ig.png/"/></a>
```
Comment (dcwalk) [inline _includes/footer.html]: any reason why png vs. svg for instagram? would be good to be consistent if possible

**Review by dcwalk — COMMENTED** [review]
Uuge!!

I took a look, but given the size/number of issues I'm not sure I've picked apart different elements to effectively review, so bear in mind I've likely missed stuff. Left some comments inline,

Some general thoughts: 
- readability feels good, tho the spacing feels a little too much around the secondary nav layour and the offset for the main body is distracting -- see screenshot 1
- alignment issue on interview page caused by offset issue -- see screenshot 2 **actually this might be on every page**
- the throws to different sections are neat! However on my comp in different displays they are quite large and spacious, and the differing line widths feel a bit visually confusing. I'm wondering if we could have a smaller image or think about layouts that don't have so much whitespace? Also, the header "Further Reading" feels out of sync with weights on the rest of the page (I think subtitle, but section weights have been strong, as below) Also it appears the EDGI square logo is not consistently aligned on pages -- see screenshot 3
- the bold on review comments is difficult to read (I know this isn't a new change, but whoa, just really dug into that section

**1**
screen shot 2017-11-28 at 11 43 33 am


**2**
screen shot 2017-11-28 at 11 55 05 am

**3**
screen shot 2017-11-28 at 11 57 06 am

**Comment by dcwalk** [comment]
also pinging @suchthis :)

**Comment by dcwalk** [comment]
Oh, and we should probably rebase #67 on this once merged in. It is super tiny, and probably shouldn't have left it so long, but there will be some minor adjustments/conflicts I think 🔀

File: _includes/footer.html (hunk 3 above)
Comment (shaqsingh) [reply _includes/footer.html]: Their branding website only had .png and an .eps that i couldn't open with my version of Illustrator

**Comment by dcwalk** [comment]
This looks awesome! @suchthis also reviewed and 👍  on slack, so I'm gonna merge in and re-deploy in the next day 🚀 🤖

**Review by dcwalk — APPROVED** [review]

**Pull request merged**